src/Analyzers/MSTest.Analyzers/Helpers/DiagnosticIds.cs

```diff
@@ -11,4 +11,5 @@ internal static class DiagnosticIds
     public const string PublicTypeShouldBeTestClassRuleId = "MSTEST0004";
     public const string TestContextShouldBeValidRuleId = "MSTEST0005";
     public const string AvoidExpectedExceptionAttributeRuleId = "MSTEST0006";
+    public const string PriorityAttributeOnTestMethodRuleId = "MSTEST00010";
```

Comment:
Please change ID, `MSTEST00010` had already dibs from my side so there'll be a collision.